io.github.nokse22.Exhibit.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id" : "io.github.nokse22.Exhibit",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "exhibit",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
```

Comment:
this uses Gtk4. Why X11 only?

Reply:
Unfortunately there is an issue between the library that I use to render the 3d models and Gtk, so it only works in X11 for some reason, still don't know how to fix it.